tests/test_container_provider.py

```diff
@@ -40,82 +52,100 @@ class AnsibleModuleFailed(Exception):
 
 @pytest.fixture()
 def miq(miq_api_class, miq_ansible_module, the_provider):
+
+    # TODO(jaryn) This would ideally be inside test cases that need it.
```

Comment:
IINM, this is possible after https://github.com/dkorn/manageiq-ansible-module/pull/15 is merged right?